test/utils/browser.js

```diff
@@ -81,6 +144,11 @@ exports.assertBrowser = function () {
   });
 };
 
+exports.simpleExecute = function (fn, args, cb) {
+  // DEV: Barebones execute function on the browser, this works when .execute does not
+  this.browser.execute(functionToString(fn).body, args, cb);
+};
```

Comment:
To make sure we are on the same page, the signature of `simpleExecute` and `execute` are not the same -- `execute` is more like a `.call` and `.simpleExecute` is more like a `.apply`:

```
simpleExecute(function (arg1, arg2) { codeGoesHere(); }, [arg1, arg2], cb);
execute(function (arg1, arg2) { codeGoesHere(); }, arg1, arg2, cb);
```